docs/content.zh/docs/dev/table/sql/queries/window-join.md

```diff
@@ -0,0 +1,192 @@
++-------------+------------------------+------+-----+--------+---------------------------------+
+|        name |                   type | null | key | extras |                       watermark |
++-------------+------------------------+------+-----+--------+---------------------------------+
+|     Time  | TIMESTAMP(3) *ROWTIME* | true |     |        |    `Time` - INTERVAL '1' SECOND |
+|     Num     | INT                    | true |     |        |                                 |
+|     Id      | STRING                 | true |     |        |                                 |
++-------------+------------------------+------+-----+--------+---------------------------------+
+
+Flink SQL> SELECT * FROM RightTable;
++------------------+-------+------+
+|    `Time`        | Num   | Id   |
++------------------+-------+------+
+| 2020-04-15 12:01 |  2    |  R2  |
+| 2020-04-15 12:04 |  3    |  R3  |
+| 2020-04-15 12:05 |  4    |  R4  |
++------------------+-------+------+
+
+Flink SQL> SELECT L.Num as L_Num, L.Id as L_Id, R.Num as R_Num, R.Id as R_Id, L.window_start, L.window_end
+           FROM (
+               SELECT * FROM TABLE(TUMBLE(TABLE LeftTable, DESCRIPTOR(`Time`), INTERVAL '5' MINUTES))
+           ) L
+           FULL JOIN (
+               SELECT * FROM TABLE(TUMBLE(TABLE RightTable, DESCRIPTOR(`Time`), INTERVAL '5' MINUTES))
+           ) R
+           ON L.Num = R.Num AND L.window_start = R.window_start AND L.window_end = R.window_end;
++--------+--------+--------+--------+---------------------+---------------------+
+|  L_Num |  L_Id  |  R_Num |  R_Id  |     window_start    |     window_end      |
++--------+--------+--------+--------+---------------------+---------------------+
+|  1     |  L1    |  null  |  null  |   2020-04-15 12:00  |  2020-04-15 12:05   |
+|  null  |  null  |  2     |  R2    |   2020-04-15 12:00  |  2020-04-15 12:05   |
+|  3     |  L3    |  3     |  R3    |   2020-04-15 12:00  |  2020-04-15 12:05   |
+|  2     |  L2    |  null  |  null  |   2020-04-15 12:05  |  2020-04-15 12:10   |
+|  null  |  null  |  4     |  R4    |   2020-04-15 12:05  |  2020-04-15 12:10   |
++--------+--------+--------+--------+---------------------+---------------------+
+```
+
+*Note: in order to better understand the behavior of windowing, we simplify the displaying of timestamp values to not show the trailing zeros, e.g. `2020-04-15 08:05` should be displayed as `2020-04-15 08:05:00.000` in Flink SQL Client if the type is `TIMESTAMP(3)`.*
+
+
+## SEMI
+Semi Window Joins returns a row from one left record if there is at least one matching row on the right side within the common window.
+
+```sql
+Flink SQL> SELECT *
+           FROM (
+               SELECT * FROM TABLE(TUMBLE(TABLE LeftTable, DESCRIPTOR(row_time), INTERVAL '5' MINUTES))
+           ) L WHERE L.Num IN (
+             SELECT Num FROM (   
+               SELECT * FROM TABLE(TUMBLE(TABLE RightTable, DESCRIPTOR(row_time), INTERVAL '5' MINUTES))
```

Comment:
row_time -> `Time` . 


Why not using a lower-case and non-reserved-keyword as the field names...?